lib/chat_models/ollama_ai_fields.ex

```diff
@@ -0,0 +1,78 @@
+defmodule LangChain.ChatModels.OllamaAIFields do
+  defmacro fields do
+    quote do
+      @receive_timeout 60_000 * 5
+
+      field :endpoint, :string, default: "http://localhost:11434/api/chat"
+
+      # Enable Mirostat sampling for controlling perplexity.
+      # (default: 0, 0 = disabled, 1 = Mirostat, 2 = Mirostat 2.0)
+      field :mirostat, :integer, default: 0
```

Comment:
For all these fields with special docs, it would be helpful to include links to the docs where these come from.